.isort.cfg

```diff
@@ -0,0 +1,15 @@
+[settings]
+combine_as_imports=1
+include_trailing_comma=1
```

Comment:
Default for `include_trailing_comma` is false, any particular reason you set it? I have a week preference against.

Reply:
Primarily to avoid 2 lines of diffs instead of one if/when we add to import lists.

I have a weak preference for the comma of hope.

Reply:
Ah, looking again I see this PR uses [multi line output mode](https://github.com/timothycrosley/isort#multi-line-output-modes) 3, which does this:

```py
from third_party import (
    lib1,
    lib2,
    lib3,
)
```

in which case that would require a two-line diff. I will express a weak preference for the much more concise mode 5 ([which Django uses](https://github.com/django/django/blob/master/setup.cfg#L16)), which does this:

```py
from third_party import (
    lib1, lib2, lib3
)
```

That being said, I defer to the person actually who does the work to open the PR and fix all of the existing code to match a consistent standard :)